components/ambient-control-plane/Dockerfile

```
@@ -0,0 +1,35 @@
+
+COPY ambient-api-server/ ./ambient-api-server/
+COPY ambient-sdk/ ./ambient-sdk/
+COPY ambient-control-plane/ ./ambient-control-plane/
+
+WORKDIR /build/ambient-control-plane
+
+RUN go mod download && \
+    CGO_ENABLED=0 GOOS=linux go build -ldflags="-s -w" \
+    -o ambient-control-plane ./cmd/ambient-control-plane
+
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)